jscomp/others/belt_Option.mli

```diff
@@ -27,6 +27,22 @@
     Utilities for option data type
 *)
 
+val applyU : 'a option -> ('a -> unit [@bs]) -> unit
+
+(** Uncurried version of [apply] *)
+
+val apply : 'a option -> ('a -> unit) -> unit
+(**
+  [apply optionValue f]
+  
+  If [optionValue] is [Some value], it calls [f value]; otherwise returns [unit]
```

Comment:
Strictly speaking, the second branch returns `()` not `unit` (since we're talking about the value level).